app/models/validations/tenancy_validations.rb

```diff
@@ -32,4 +32,10 @@ def validate_fixed_term_tenancy(record)
   def validate_other_tenancy_type(record)
     validate_other_field(record, 3, :tenancy, :tenancyother)
   end
+
+  def validate_joint_tenancy(record)
+    if record.hhmemb == 1 && record.joint != 2 && record.collection_start_year == 2022
```

Comment:
Should this be `record.collection_start_year >= 2022`?

Reply:
Good catch

Reply:
I hadn't considered future forms